src/redux/actions.js

```diff
@@ -0,0 +1,11 @@
+export const ACTION = "ACTION";
```

Comment:
You would give the action a better name.
This will be an issue, if you add more actions. 

Example
```suggestion
export const COUNT_UP = "COUNT_UP";
```